0004-rfcs-over-adr-pdr/0004-rfcs-over-adr-pdr.md

```diff
@@ -0,0 +1,9 @@
+# RFC 0004 - RFCs over ADRs and PDRs
+
+This RFC describes the relationship between RFCs, ADRs, and PDRs.
+
+'ADR' refers to Architecture Decision Record, and 'PDR' refers to Product Decision Record. These records, as well as their processes, are currently located in the private Giant Swarm Intranet.
+
+ADRs and PDRs should be superseded by RFCs. This allows for one public process for discussing and recording ideas.
+
```

Comment:
not sure about this. IMO ADRs/PDRs and RFCs serve different purposes. IF ADRs/PDRs are the records about decisions we made and committed to, RFCs can contain any ideas and there is no guarantee (and also no process atm) that they will become reality at some point.

I see RFCs as a 'start of conversation', to ask for input, to get feedback. 
If am wrong and don’t understand the meaning of RFCs then we need to establish a process for RFC lifecycle. Because getting few approvals is definitely not enough to take a decision.